.devcontainer/post-create.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+# Add welcome message
+sudo cp .devcontainer/welcome.txt /usr/local/etc/vscode-dev-containers/first-run-notice.txt
+
+# Switch to vets-api ruby version
+export PATH="${HOME}/.asdf/shims:${HOME}/.asdf/bin:${PATH}"
+asdf install ruby $( cat .ruby-version )
+asdf global ruby $( cat .ruby-version )
+
+git clone https://github.com/department-of-veterans-affairs/vets-api-mockdata.git ../vets-api-mockdata
+
+sudo apt update
+sudo apt install -y libpq-dev pdftk shared-mime-info postgresql-15-postgis-3
+
+gem install bundler
+NUM_CORES=$( cat /proc/cpuinfo | grep '^processor'|wc -l )
+bundle config --global jobs `expr $NUM_CORES - 1`
+
+# Update test DB config
+echo 'test_database_url: postgis://postgres:password@localhost:5432/vets_api_test?pool=4' > config/settings/test.local.yml
+
+# Add service config
+if [ ! -f config/settings.local.yml ]; then
+  cp config/settings.local.yml.example config/settings.local.yml
+  cat <<EOT >> config/settings.local.yml
+database_url: postgis://postgres:password@localhost:5432/vets_api_development?pool=4
+
+redis:
+  host: localhost
+  port: 6379
+  app_data:
+    url: redis://localhost:6379
+  sidekiq:
+    url: redis://localhost:6379
+
+betamocks:
+  cache_dir: ../vets-api-mockdata
+
+# Allow access from localhost and shared github URLs.
+virtual_hosts: ["127.0.0.1", "localhost", !ruby/regexp /.*\.app\.github\.dev/]
```

Comment:
If you view `/v0/status` you get an error which raises on [this line](https://github.com/department-of-veterans-affairs/vets-api/tree/master/app/controllers/concerns/headers.rb#L11).

Reply:
Another good catch, thanks!

I was looking into updating `app/controllers/concerns/headers.rb` to be able to handle the regex, but it occurs to me that I don't know if raising that exception is necessary at all:

It was [added in 2018](https://github.com/department-of-veterans-affairs/vets-api/commit/092efbcce3af3f05f186ae28a24851606e0f0e36), well before [rails was upgraded to version 6 in 2020](https://github.com/department-of-veterans-affairs/vets-api/commit/4a86bec0d31075f96dd118772a6b2cdd5a51aa89). The [HostAuthorization middleware was added in Rails 6](https://edgeguides.rubyonrails.org/6_0_release_notes.html#action-pack-notable-changes).

Do you see any reason not to just remove `block_unknown_hosts`?

Reply:
That seems reasonable. 👍

Reply:
I was looking at how to test this, but:

a) seems specs are generally meant to test controllers without middleware loaded, and
b) we would be duplicating the ActionDispatch tests

Since vets-api is fronted by the TIC and the revproxy, neither of which will send it requests with incorrect host headers, do we need a test for this functionality? Seems like it would be more in the realm of a smoke test anyway.

Reply:
I think it's good.